apps/web/src/services/dashboards/dashboard-customize/modules/VariableSelectorDropdown.vue

```diff
@@ -184,10 +187,17 @@ const handleUpdateSearchText = debounce((text: string) => {
     reloadMenu();
 }, 200);
 
-const loadSearchResourceOptionsByCostAnalysis = async () => {
+const loadSearchResourceOptions = async () => {
     if (state.variableProperty.options?.type === 'SEARCH_RESOURCE') {
+        // NOTE: Some variables(asset) require specific API filters.
+        const apiQueryHelper = new ApiQueryHelper();
+        if (state.variableProperty.name === ASSET_REFERENCE_TYPE_INFO.asset_compliance_type.name) {
+            apiQueryHelper.addFilter({ k: 'labels', o: '=', v: 'Compliance' });
+        } else if (state.variableProperty.name === ASSET_REFERENCE_TYPE_INFO.asset_account.name) {
+            apiQueryHelper.addFilter({ k: 'provider', o: '=', v: 'aws' });
+        }
```

Comment:
description: This code is for this.

![스크린샷 2023-06-09 오후 5 23 10](https://github.com/cloudforet-io/console/assets/18563857/527f06c8-d74f-4a4f-b26e-3ceed9d76262)

Reply:
Thanx for explanation!